passes/techmap/abc_new.cc

```diff
@@ -38,7 +38,8 @@ std::vector<Module*> order_modules(Design *design, std::vector<Module *> modules
 				sort.edge(submodule, m);
 		}
 	}
-	log_assert(sort.sort());
+	bool sorted = sort.sort();
```

Comment:
Given that `sort.sorted` is the sorted list, I think calling the return from `sort.sort()` `sorted` is misleading and it should instead be something like `is_sorted`.